docs/03-tutorials/00-serverless/svls-15-use-external-scalers.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Use external scalers
+---
+
+This tutorial shows how to use external resource scaler (like HorizontalPodAutoscaler or Keda's ScaledObject) with the Serverless Function.
+
+Keep in mind that serverless functions implements the [scale-subresource](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource) which mean that you can use every `kubernetes-based` scaler instead of these described in this tutorial.
```

Comment:
```suggestion
Keep in mind that the Serverless Functions implement the [scale-subresource](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource), which means that you can use any Kubernetes-based scaler.
```
Since we mentioned that any Kubernetes-based scaler can be used, I guess we can drop the second part of the sentence.